### PR TITLE
Add ALESCo meeting minutes, 2024-08-21

### PR DESCRIPTION
**Pull request by jonathanspw**

### CODE REVIEW

File: docs/alesco/meeting-minutes/2024-08-21.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+## Decisions Adopted
+1. SPICE support to be added back to AlmaLinux 10
+   - Removed in RHEL 9
+   - Will revisit for enablement in AlmaLinux 9 at a later time
+   - Unanimous decision
+
+## Minutes
+- SPICE Support in AlmaLinux
+  - Neal: Dropped in RHEL due to no RHEL stacks dependent on it.
```
Comment (codyro) [inline docs/alesco/meeting-minutes/2024-08-21.md:29]: Mind adding the rhbz reference I pulled for this decision too? So people can see full context.

Comment (codyro) [reply docs/alesco/meeting-minutes/2024-08-21.md:29]: https://bugzilla.redhat.com/show_bug.cgi?id=2030592#c4

Comment (jonathanspw) [reply docs/alesco/meeting-minutes/2024-08-21.md:29]: Still got that link handy?

Comment (codyro) [reply docs/alesco/meeting-minutes/2024-08-21.md:29]: ^

Comment (Conan-Kudo) [reply docs/alesco/meeting-minutes/2024-08-21.md:29]: Note that SPICE supports VP9 (which addresses the issue about H264): https://gitlab.freedesktop.org/spice/spice/-/blob/master/server/gstreamer-encoder.c#L845-862

File: docs/alesco/meeting-minutes/2024-08-21.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- Andrew: Secure-Boot Signing ELRepo or other 3rd party kernels
+  - Building works, signing works, never got further feedback on if we want to do this or not
+  - Are we going to collaborate with 3rd party if we were to do this?
+  - Neal: Anything we sign should be AlmaLinux-hosted
+    - All: agreed on this point
+  - Neal: Have discussed with Jonathan about also maintaing the Hyperscale kernel for AlmaLinux
+  - Jonathan: Situation is too fluid right now to make a decision on this at the moment.
+  - Jonathan: Better if we have a SIG that wants to maintain something within AlmaLinux versus pulling from a 3rd party
+  - Neal: Signing SIG kernels in general
+    - We have to be able to trust the full tree and trace back all chances to the origin
```
Comment (Conan-Kudo) [inline docs/alesco/meeting-minutes/2024-08-21.md:74]: `chances` -> `changes`?

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]

**Review by bennyvasquez — APPROVED** [review]

**Review by Conan-Kudo — APPROVED** [review]

**Review by bathomas — APPROVED** [review]

File: docs/alesco/meeting-minutes/2024-08-21.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  - Ben: Already has requests within UCL for re-adding support for QLogic cards
+  - Alex (Board): Must be careful when enabling things that might appear easy to flip a flag now but will potentially later become more difficult to maintain.
+  - Jonathan: We can agree on an initial list of things to re-enable that are very common and/or we have direct experience with and vote on the rest as demand arises throughout the AlmaLinux 10 life cycle.
+- Elkhan: Separate kernel for PPC with KVM enabled
+  - Neal: Basically what centos hyperscale does with their kernel - RHEL-ish config (based on Fedora) + enablement on some thingfs
+  - Neal: What if we just re-enable it in the main Alma kernel since it is a flag?
+    - Jonathan: We can mark that feature specifically as experimental, even in the main Alma kernel
+    - Andrew: Just changing the flag isn't enough, a few patches are required to get it to build
+  - Elkhan: Use-case is OpenQA and image building
+    - All: not aware of any use-cases or requests from users
```
Comment (LKHN) [inline docs/alesco/meeting-minutes/2024-08-21.md:62]: We may expect OpenStack as one possible use-case/request.
Currently, we are using Power based OpenStack instance provided by [OSU Open Source Lab](https://osuosl.org/).

Feel free to decide whether need to add this part or not. Just shared my thoughts here.

**Review by LKHN — CHANGES_REQUESTED** [review]

**Pull request merged**